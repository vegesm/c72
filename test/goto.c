main () {
    int a[1];

    goto skip1;
    printf("test1 - fail\n");
    skip1:
        printf("test1 - ok\n");

    goto (1 ? skip2 : 0);
    printf("test2 - fail\n");
    skip2:
        printf("test2 - ok\n");

    a[0] = skip3;
    goto a[0];
    printf("test3 - fail\n");
    skip3:
        printf("test3 - ok\n");
}