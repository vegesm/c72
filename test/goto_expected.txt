test1 - ok
test2 - ok
test3 - ok
